app/src/main/res/layout-land/state_fragment.xml

```diff
@@ -67,21 +67,25 @@
         android:orientation="vertical"
         tools:layout_constraintGuide_percent="0.5f"
         app:layout_constraintGuide_percent="@{viewModel.centerGuidelinePercentage.floatValue()}" />
-    </androidx.constraintlayout.widget.ConstraintLayout>
-
-    <FrameLayout
-      android:id="@+id/audio_fragment_container"
-      android:layout_width="match_parent"
-      android:layout_height="wrap_content"
-      android:layout_gravity="top"
-      android:layout_marginStart="24dp">
 
       <FrameLayout
-        android:id="@+id/audio_fragment_placeholder"
-        android:layout_width="match_parent"
+        android:id="@+id/audio_fragment_container"
```

Comment:
we can reformat this file.

Reply:
Yes, I avoided this earlier just to keep PR review easy. Will now make a commit to fix all reformatting.